modules/dockerregistry/dockerregistry.go

```diff
@@ -0,0 +1,96 @@
+package dockerregistry
+
+import (
+	"context"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// DockerRegistryContainer represents the DockerRegistry container type used in the module
+type DockerRegistryContainer struct {
+	testcontainers.Container
+}
+
+// RunContainer creates an instance of the DockerRegistry container type
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*DockerRegistryContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        "registry:2",
+		ExposedPorts: []string{"5000:5000/tcp"},
```

Comment:
I think we do not want to always force a fixed port for the container:
```suggestion
		ExposedPorts: []string{"5000/tcp"},
```